scripts/run_benches_for_runtime.sh

```diff
@@ -44,32 +47,42 @@ pallets=(
 	pallet-asset-switch
 )
 
-
-
 # Add Peregrine-only pallets here!
 if [ "$runtime" = "peregrine" ]; then
   pallets+=(
 	pallet-sudo
   )
 fi
 
-echo "[+] Running all runtime benchmarks for $runtime --chain=$chain"
+echo "[+] Running all runtime benchmarks for \"$runtime\", \"--chain=$chain\" and profile \"$profile\""
 
 cargo build $standard_args
 
+if [ $profile == "dev" ]; then
+    target_folder="debug"
+	# We care about benchmark correctness, not accuracy.
+	additional_args="--steps=2 --repeat=1 --default-pov-mode=ignored --no-verify"
+else
+    target_folder=$profile
+	additional_args="--header=\"HEADER-GPL\" --template=\".maintain/runtime-weight-template.hbs\" --output=\"./runtimes/${runtime}/src/weights/\""
```

Comment:
steps=50 and repeat=20 are the default values right? Otherwise, they are missing here

Reply:
Yes, that's why I removed them.